misago/core/migrations/0002_basic_settings.py

```diff
@@ -33,7 +33,7 @@ def create_basic_settings_group(apps, schema_editor):
                 {
                     'setting': 'forum_index_title',
                     'name': _("Index title"),
-                    'description': _("You may set custon title on forum index by typing it here."),
+                    'description': _("You may set a custom title on forum index by typing it here."),
```

Comment:
Hey,

Sadly, its not that easy to just fix a typo in settings migration. You'll need to create new migration that runs `migrate_settings_group` with data structure containing current settings group (including corrected spelling) and remove the `_()` from around the strings in old migration.

You'll also need to regenerate language files for english language for things to stay in sync for translators.

If you want, I may do this myself in coming days to show you (and others) how settings migrations should be fixed.